modules/integration_azure-api-management-service/variables.tf

```
@@ -0,0 +1,177 @@
+# Module specific
+
+# Heartbeat detector
+
+variable "heartbeat_disabled" {
+  description = "Disable all alerting rules for heartbeat detector"
+  type        = bool
+  default     = null
+}
+
+variable "heartbeat_notifications" {
+  description = "Notification recipients list per severity overridden for heartbeat detector"
+  type        = map(list(string))
+  default     = {}
+}
+
+variable "heartbeat_timeframe" {
+  description = "Timeframe for heartbeat detector (i.e. \"10m\")"
+  type        = string
+  default     = "20m"
+}
+
+variable "heartbeat_aggregation_function" {
+  description = "Aggregation function and group by for heartbeat detector (i.e. \".mean(by=['host'])\")"
+  type        = string
+  default     = ".mean(by=['azure_resource_name', 'azure_resource_group_name', 'azure_region'])"
+}
+
+# capacity detector
+
+variable "capacity_notifications" {
+  description = "Notification recipients list per severity overridden for capacity detector"
+  type        = map(list(string))
+  default     = {}
+}
+
+variable "capacity_aggregation_function" {
+  description = "Aggregation function and group by for capacity detector (i.e. \".mean(by=['host'])\")"
+  type        = string
+  default     = ".mean(by=['azure_resource_name', 'azure_resource_group_name', 'azure_region'])"
+}
+
+variable "capacity_transformation_function" {
+  description = "Transformation function for capacity detector (i.e. \".mean(over='5m')\")"
+  type        = string
+  default     = ".min(over='5m')"
+}
+
+variable "capacity_disabled" {
+  description = "Disable all alerting rules for capacity detector"
+  type        = bool
+  default     = null
+}
+
+variable "capacity_disabled_critical" {
+  description = "Disable critical alerting rule for capacity detector"
+  type        = bool
+  default     = null
+}
+
+variable "capacity_disabled_major" {
+  description = "Disable major alerting rule for capacity detector"
+  type        = bool
+  default     = null
+}
+
+variable "capacity_threshold_critical" {
+  description = "Critical threshold for capacity detector (in GB)"
+  type        = number
+  default     = 90
+}
+
+variable "capacity_threshold_major" {
+  description = "Major threshold for capacity detector (in GB)"
+  type        = number
+  default     = 80
```

```suggestion
  default     = 90
```
An API Management service working at 80% of its capacity is a well sized resource in my opinion. :)